plugins/ScheduledReports/Controller.php

```diff
@@ -26,7 +26,11 @@ public function index()
         $view = new View('@ScheduledReports/index');
         $this->setGeneralVariablesView($view);
 
-        $view->countWebsites = count(APISitesManager::getInstance()->getSitesIdWithAtLeastViewAccess());
+        $siteTimezone = $this->site->getTimezone();
+        $dateTimeZone = new \DateTimeZone($siteTimezone);
+
+        $view->timeZoneDifference = -$dateTimeZone->getOffset(new \DateTime());
+        $view->countWebsites      = count(APISitesManager::getInstance()->getSitesIdWithAtLeastViewAccess());
```

Comment:
If so that would have been there before, as well. I've only added the detection for timezone offset.